tools/docker/docker-compose-dev.yaml

```diff
@@ -162,10 +166,14 @@ services:
       POSTGRESQL_PASSWORD: secret
       POSTGRESQL_ADMIN_PASSWORD: secret
       POSTGRESQL_DATABASE: eda
+      POSTGRES_HOST_AUTH_METHOD: trust
     ports:
       - '5432:5432'
     volumes:
       - 'postgres_data:/var/lib/pgsql/data'
+      - './postgres_ssl_config/postgresql-cfg:/opt/app-root/src/postgresql-cfg:Z'
+      - './postgres_ssl_config/certs:/opt/app-root/src/certs:Z'
```

Comment:
@msmagnanijr should this be a ENV VAR since the paths are going to be different for each user

Reply:
@Alex-Izquierdo Exactly. I followed the doc image: https://github.com/sclorg/postgresql-container/tree/62a0e88ae0e5f19c3dad1cb6fea3778dfa2dbf2f/examples/enable-ssl